Mathlib/RingTheory/Finiteness/Defs.lean

```diff
@@ -103,9 +103,9 @@ variable (R A B M N : Type*)
 
 /-- A module over a semiring is `Module.Finite` if it is finitely generated as a module. -/
 protected class Module.Finite [Semiring R] [AddCommMonoid M] [Module R M] : Prop where
-  out : (⊤ : Submodule R M).FG
+  finite : (⊤ : Submodule R M).FG
```

Comment:
Arguably this should be `fg` or `fg_top`

Reply:
I did think this but wasn't sure. Given the upvotes I'll change it.

Reply:
OK it's `fg_top` now which I think is better.